libraries/chain/protocol/asset_ops.cpp

```diff
@@ -45,9 +45,6 @@ bool is_valid_symbol( const string& symbol )
     if( !isalpha( symbol.front() ) )
         return false;
 
-    if( !isalpha( symbol.back() ) )
-        return false;
-
```

Comment:
Still better to check here.

Reply:
done at the move check commit https://github.com/bitshares/bitshares-core/pull/705/commits/2f33311603dbcdb031776667140ff48770328391